tools/Directory.Packages.props

```diff
@@ -2,8 +2,12 @@
   <Import Project="..\Directory.Packages.props" />
   <ItemGroup>
     <PackageVersion Include="McMaster.Extensions.CommandLineUtils" Version="4.1.1" />
-    <PackageVersion Include="Microsoft.Build" Version="17.6.3" />
-    <PackageVersion Include="NuGet.ProjectModel" Version="6.11.0" />
+    <PackageVersion Include="Microsoft.Build" Version="17.7.2" />
```

Comment:
For now, it is the latest version officially supporting .NET7

Reply:
Maybe worth adding that as a comment in the file?

Reply:
I would keep as is, we will be removing this support within ~month.